### PR TITLE
initial skeleton

### PR DESCRIPTION
**Pull request by m00sey**
based off issuer

### CODE REVIEW

File: pkg/didcomm/verifier/cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+		return nil, errors.Wrap(err, "unmarshal inbound.amqp")
+	}
+
+	ariesSub := r.vp.Sub("aries")
+	vdris, err := context.GetAriesVDRIs(ariesSub)
+	if err != nil {
+		return nil, errors.Wrap(err, "GetAriesVDRIs")
+	}
+
+	amqpInbound, err := amqp.NewInbound(config.Endpoint(), external, "present-proof", "", "")
```
Comment (pfeairheller) [inline pkg/didcomm/verifier/cmd/root.go:189]: This makes me very happy

File: pkg/didcomm/verifier/cmd/root_test.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*
```
Comment (pfeairheller) [inline pkg/didcomm/verifier/cmd/root_test.go:1]: I'm leaning towards thinking we don't need tests for the cmd stuff...  thoughts?

File: pkg/ursa/verifier.go [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+type VerifierServer struct {
+}
+
+func NewVerifier() *VerifierServer {
+	return &VerifierServer{}
+}
+
+func (r *VerifierServer) SendRequestPresentation(msg *ppprotocol.RequestPresentation, myDID, theirDID string) (string, error) {
+	return "", nil
```
Comment (pfeairheller) [inline pkg/ursa/verifier.go:25]: Nailed it!

**Review by pfeairheller — APPROVED** [review]

File: pkg/didcomm/verifier/cmd/root_test.go (hunk 2 above)
Comment (m00sey) [reply pkg/didcomm/verifier/cmd/root_test.go:1]: open another issue to address this, but yeah.

**Pull request merged**